massa-pos-exports/src/pos_final_state.rs

```diff
@@ -245,8 +319,11 @@ impl PoSFinalState {
     }
 
     /// Feeds the selector targeting a given draw cycle
-    fn feed_selector(&self, draw_cycle: u64) -> PosResult<()> {
+    pub fn feed_selector(&self, draw_cycle: u64) -> PosResult<()> {
```

Comment:
if this can be pub(crate), make it so.

Reply:
I have to use it when interpolating downtime in final_state, so I sadly can't